fuzzing/src/targets/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ add_library(fuzztargets
 
 target_compile_definitions(fuzztargets
   PRIVATE
-  "${LOGGER_NAME}")
+  "${LOGGER_NAME}" HAVE_ARCHIVE)
```

Comment:
if you don't mind let's put `HAVE_ARCHIVE` into a new line; it's a bit of a guess but FreeType's code is usually quite vertically structured